includes/view/Rooms_view.php

```diff
@@ -53,7 +53,19 @@ function Room_view($room, ShiftsFilterRenderer $shiftsFilterRenderer, ShiftCalen
     return page_with_title(glyph('map-marker') . $room['Name'], [
         $assignNotice,
         $description,
-        tabs($tabs, $selected_tab)
+        auth()->can('admin_rooms') ? buttons([
+            button(
+                page_link_to('admin_rooms', ['show' => 'edit', 'id' => $room->id]),
```

Comment:
$room->id is still in array here, but that is no problem when merging the room model rewrite.